### DISCUSSION TITLE
如果节点数据很多是否支持停止当前graph的渲染？

### DISCUSSION THREAD

**Discussion by hshuaibin** [antvis/X6#1729] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/45307029/148343699-0c663b51-66a5-4c31-a9df-6eefe76e07e8.png)

![image](https://user-images.githubusercontent.com/45307029/148343801-b929ac37-06aa-44ca-a89d-a7916f075dbe.png)

startBatch、stopBatch，尝试使用开始事务和终止事务方法，然后手动触发stopBatch方法，但是此时还是将事务执行完成，是否可以主动停止整个事务，使其节点不渲染到graph上

**Comment by NewByVector** [comment, marked as answer]
不支持中途停止渲染的。
